### ISSUE TITLE
Fixed position of GMap?

### ISSUE THREAD

**Issue by rannieo** [xkjyeah/vue-google-maps#344]
Is it possible to make a fixed center or relative in GMap?

**Comment by abrhambas01** [comment]
@einnar82  yes. just

<gmap-map ref="map" :center="{ lat : 43.2000 lng :43.400 }</gmap-map>

**Comment by rannieo** [comment]
yeah, it will center, i want is if i drag to other place, it will back to the center

**Comment by abrhambas01** [comment]
Which center ? where the marker is currently located?

**Comment by rannieo** [comment]
center of the map

**Comment by abrhambas01** [comment]
this.$refs.map.panTo({
				lat : this.center.lat,
				lng : this.center.lng 
			});

Something like that
